elyra/pipeline/processor_local.py

```diff
@@ -0,0 +1,107 @@
+    same way when processed by this processor. Also, any relationship or specific capabilities
+    associated with a particular runtime is not supported by local mode and any additional properties
+    other then the specific file to be executed is ignored by the local processor.
+
+    Note: Execution happens in-place and a lodger of runs are available at $TMPFILE/elyra/pipeline-name-<timestamp>
+    """
+    _type = 'local'
+
+    def __init__(self, root_dir=None):
+        if root_dir:
+            self._root_dir = root_dir
+
+    @property
+    def type(self):
+        return self._type
+
+    def process(self, pipeline):
+        timestamp = datetime.now().strftime('%m%d%H%M%S')
+        pipeline_name = f'{pipeline.name}-{timestamp}'
+
+        pipeline_work_dir = self._get_pipeline_work_dir(pipeline_name)
+        self.log.info(f'Processing Pipeline : {pipeline.name} at {pipeline_work_dir}')
+
+        # TODO: consider ordering
+        for operation in pipeline.operations.values():
+            notebook = self.get_absolute_path(operation.filename)
+            assert os.path.isfile(notebook), "Invalid '{}'".format(notebook)
+
+            self._execute_file(pipeline_work_dir, notebook)
+
+        object_storage_path = f'{pipeline_work_dir}'
+        return PipelineProcessorResponse('', '', object_storage_path)
+
+    def export(self, pipeline, pipeline_export_format, pipeline_export_path, overwrite):
+        raise NotImplementedError('Local pipelines does not support export functionality')
+
+    def _get_pipeline_work_dir(self, pipeline_name):
+        log_dir = os.path.join(tempfile.tempdir, 'elyra', pipeline_name)
+        if not os.path.exists(log_dir):
+            os.mkdir(log_dir)
+        return log_dir
+
+    def _execute_file(self, work_dir, filename):
+        notebook_dir = os.path.dirname(filename)
+        notebook_name = os.path.basename(filename)
+        stdout = os.path.join(work_dir, f'{notebook_name}.out')
+        stderr = os.path.join(work_dir, f'{notebook_name}.err')
+
+        self.log.debug(f'Processing: {filename}')
+        with open(stdout, 'w') as stdout_file, open(stderr, 'w') as stderr_file:
+            t0 = time.time()
+            try:
+                # Try executing with configured python kernel
+                papermill.execute_notebook(
+                    input_path=filename,
+                    output_path=filename,
+                    stdout_file=stdout_file,
+                    stderr_file=stderr_file,
+                    cwd=notebook_dir
+                )
+            except NoSuchKernel:
+                # force default python kernel
+                papermill.execute_notebook(
+                    input_path=filename,
+                    output_path=filename,
+                    stdout_file=stdout_file,
+                    stderr_file=stderr_file,
+                    cwd=notebook_dir,
+                    kernel_name="python3"
```

Comment:
Actually, per the large comment for the class, this shouldn't happen.  In addition, it will fail for non-python kernels anyway.  As stated above: _If a notebook doesn't run using `run all` it will fail in the same way when processed by this processor._ so we shouldn't make any attempts to run the notebook on any error.

Reply:
I created an issue and will solve this problem more globally as it's the same current behavior as in the kfp runtime

Reply:
Then please remove this error handling.  It should not be done.

Reply:
I would like to have consistent behavior, so will remove when addressing the other issue if that is ok.

Reply:
I don't really understand that.  The "other issue" is in a completely different repository and this is an error case that no one should rely on.